.github/custom-lint.R

```diff
@@ -24,7 +24,10 @@ doubleQuotesLinter = function(sourceFile) {
           type = 'style',
           message = 'Only use single-quotes.',
           line = line,
-          ranges = list(c(col1, col2)))})})}
```

Comment:
Nice

Reply:
Is this the linter that can reformat code in whatever style we'd like?

Reply:
`lintr` doesn't reformat code, but it can detect certain kinds of style issues. I've revised the script based on our discussions on the style guide in Notion. There's a separate package called `styler`, but I've not used it.